### ISSUE TITLE
Add PyCheckerBear

### ISSUE THREAD

**Issue by khanchi97** [coala/coala-bears#2151]
http://pychecker.sourceforge.net/

**Comment by RaiVaibhav** [comment]
I got confused by reading `Wrapping Up` https://www.blog.pythonlibrary.org/2012/06/13/pyflakes-the-passive-checker-of-python-programs/

 `PyCheckerBear` is not fast if we compare it to `PyFlakes` but more accurate than `PyFlakes`(which currently `coala` has)

Easy Comapare https://www.blog.pythonlibrary.org/2011/01/26/pychecker-python-code-analysis/

**Comment by khanchi97** [comment]
Well, @RaiVaibhav I think it depends on the users.let say If someone wants to use `Pyflakes` or `Pylint` then I think these linters should be available for him We can not say to user that "`Pyflakes` is almost doing same as `Pylint` so you can use `Pylint`" Acc. to me it can be the case 😉

**Comment by Makman2** [comment]
Once [aspects](https://github.com/coala/cEPs/blob/master/cEP-0010.md) will arise, this redundancy will be cleaned up :+1:

**Comment by khanchi97** [comment]
@Makman2 Can you please assign me to this issue.

**Comment by yash-nisar** [comment]
Is corobo still down ? 😮

**Comment by khanchi97** [comment]
Yes corobo is still down 😞 .When we are in need of something and we couldn't get help of that then it feels how much does it matter 😉

**Comment by AkshJain99** [comment]
we have pylint bear for linting python code, why need pycheck bear?

**Comment by AkshJain99** [comment]
and also if no one is working on this issue i would like to take this up , I know the solution to add this bear 
@li-boxuan  or @Makman2  if you assign this to me i will open pr for this now only

**Comment by chay2199** [comment]
> we have pylint bear for linting python code, why need pycheck bear?

@AkshJain99 Your doubt has already been discussed in the above thread. Please have a look.

**Comment by AkshJain99** [comment]
@li-boxuan i have created a commit as soon as you assign me this issue i will open a pr
thanks

**Comment by AkshJain99** [comment]
Thanks
